third_party/move/move-vm/runtime/src/loader/mod.rs

```diff
@@ -1723,12 +1265,15 @@ impl<'a> Resolver<'a> {
     // Type resolution
     //
 
-    pub(crate) fn get_struct_type(&self, idx: StructDefinitionIndex) -> Type {
+    pub(crate) fn get_struct_type(&self, idx: StructDefinitionIndex) -> PartialVMResult<Type> {
```

Comment:
This name is confusing because StructType is a different type..

Reply:
+1 (Perhaps get_type). Or rename StructType

Reply:
Maybe a wrong place (and time) for this comment - but a type named Type is a major`code smell`. Is this MoveType?